### ISSUE TITLE
Add support for SDA virtual network IP pool

### ISSUE THREAD

**Issue by danischm** [CiscoDevNet/terraform-provider-catalystcenter#28]

**Comment by danischm** [comment]
Yes, I think that is the equivalent resource in the dnacenter provider.

**Comment by kuba-mazurkiewicz** [comment]
Resource name is: catalystcenter_virtual_network_ip_pool, it will be included in 0.1.6 version: https://github.com/CiscoDevNet/terraform-provider-catalystcenter/blob/main/CHANGELOG.md

Example:
https://github.com/CiscoDevNet/terraform-provider-catalystcenter/blob/main/examples/resources/catalystcenter_virtual_network_ip_pool/resource.tf

We are still working on this release, so if you want to test catalystcenter_virtual_network_ip_pool, you can clone this repo and build the provider using go install command

**Comment by danischm** [comment]
Added in v0.1.6 .
